google/cloud/forseti/common/util/parser.py

```diff
@@ -59,17 +59,17 @@ def format_timestamp(timestamp_str, datetime_formatter):
         str: The formatted, stringified timestamp.
     """
     try:
-        if '"' in timestamp_str or "'" in timestamp_str:
+        if '\'' in timestamp_str or '\'' in timestamp_str:
             # Make sure the timestamp is not surrounded by any quotes
-            timestamp_str = timestamp_str.replace("'")
-            timestamp_str = timestamp_str.replace('"')
+            timestamp_str = timestamp_str.replace('\'')
```

Comment:
Same here, one of the statements should be be replacing double quote.

Reply:
I think this is actually a bug, it should be `timestamp_str.replace('"', '')`

Reply:
done